xbmc/addons/AddonDatabase.cpp

```diff
@@ -845,7 +845,7 @@ bool CAddonDatabase::Search(const std::string& search, VECADDONS& addons)
     while (!m_pDS->eof())
     {
       AddonPtr addon;
-      GetAddon(m_pDS->fv(0).get_asString(),addon);
+      GetAddon(m_pDS->fv(0).get_asInt(),addon);
```

Comment:
Space needed after `,`. You could also run git-clang-format once you have add the files to be committed and it will do the fixes for you. Then just add them again and commit.

Reply:
Oh, I just keeping it as it was.
So, I'm ok to clean up lines I touch?

Reply:
Correct, you should clean only the lines you have touched.

Reply:
I dont have git-clang-format (nothing about that in docs)
looks like just a python file that i pop in PATH:
https://raw.githubusercontent.com/llvm-mirror/clang/master/tools/clang-format/git-clang-format

Reply:
Once you have clang-format installed that's all it is on top of it.